AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -2196,6 +2247,14 @@ - (BOOL)shouldBypassEnsureDisplay
 #pragma mark - For Subclasses
 
 - (ASLayout *)calculateLayoutThatFits:(ASSizeRange)constrainedSize
```

Comment:
Should `layoutThatFits` and `calculateLayoutThatFits` variances ever return nil? We may want to add `_Nonnull` annotation to these method signatures. Maybe declare `_calcualtedDisplayNodeLayout` `_Nonnull` as well.

@rahul-malik will be happy :)